src/TodoList.jsx

```diff
@@ -0,0 +1,30 @@
+const todoList = [
+    {
+      id: 1,
+      title: 'Complete week 1 assignment'
+    },
+    {
+      id: 2,
+      title: 'Complete week 2 assignment'
+    },
+    {
+      id: 3,
+      title: 'Complete week 3 assignment'
+    }
+  ];
+
+const TodoList = () => {
+    return (
+        <>
```

Comment:
The same as with the `form` tag -  `ul` tag also acts as a container. This is not a mistake though - this is optional, to make code cleaner.